invisible_cities/reco/corrections.py

```diff
@@ -9,6 +9,15 @@
 from .. evm.ic_containers       import Measurement
 
 
+opt_nearest = {"interp_method": "nearest"}
```

Comment:
Since the code below does not depend on these global variables and they serve mostly as documentation I guess this is ok. I'll tag @jacg just in case he has serious complaints. Otherwise I guess we can let this be.

Reply:
Does removing these variables cause any tests to fail? If not, then get rid of them: even if they do serve a purpose of documentation (which I can't see at first glance), without tests depending on it, it will diverge from reality sooner or later, and documentation which does not reflect reality is evil.

But do note that just because it'not used in the code below ... well this is a module's global scope so its purpose could well be for someone to import it and use it elsewhere.

Run the tests, and then apply the principle **if it's not tested, then it doesn't exist**.

If removing it doesn't break any test in the test suite, then get rid of it.

The same applies to your earier comment about the nested brackets.

Reply:
Those variables are there as a shortcut for "frequently used options", as described in the commit. The tests depend on them as they are used to specify which normalization option is used within the tests.

> its purpose could well be for someone to import it and use it elsewhere.

That is its other functionality, yes. To be used in the same way as described above.